src/Databases/DatabaseReplicatedSettings.h

```diff
@@ -7,13 +7,24 @@ namespace DB
 
 class ASTStorage;
 
-#define LIST_OF_DATABASE_REPLICATED_SETTINGS(M) \
-    M(Float, max_broken_tables_ratio, 0.5, "Do not recover replica automatically if the ratio of staled tables to all tables is greater", 0) \
+#define CONTEMPORARY_DATABASE_REPLICATED_SETTINGS(M) \
+    M(Float,  max_broken_tables_ratio, 0.5, "Do not recover replica automatically if the ratio of staled tables to all tables is greater", 0) \
     M(UInt64, max_replication_lag_to_enqueue, 10, "Replica will throw exception on attempt to execute query if its replication lag greater", 0) \
     M(UInt64, wait_entry_commited_timeout_sec, 3600, "Replicas will try to cancel query if timeout exceed, but initiator host has not executed it yet", 0) \
-    M(String, cluster_username, "default", "Username to use when connecting to hosts of cluster", 0) \
-    M(String, cluster_password, "", "Password to use when connecting to hosts of cluster", 0) \
-    M(Bool, cluster_secure_connection, false, "Enable TLS when connecting to hosts of cluster", 0) \
+    M(String, collection_name, "", "A name of a collection defined in server's config where all info for cluster authentication is defined", 0) \
+
+#define MAKE_OBSOLETE_REPLICATED_DATABASE_SETTING(M, TYPE, NAME, DEFAULT) \
+    M(TYPE, NAME, DEFAULT, "Obsolete setting, does nothing.", BaseSettingsHelpers::Flags::OBSOLETE)
+
+#define OBSOLETE_DATABASE_REPLICATED_SETTINGS(M) \
+    /** Obsolete settings that do nothing but left for compatibility reasons. Remove each one after half a year of obsolescence. */ \
+    MAKE_OBSOLETE_REPLICATED_DATABASE_SETTING(M, String, cluster_username, "") \
+    MAKE_OBSOLETE_REPLICATED_DATABASE_SETTING(M, String, cluster_password, "") \
+    MAKE_OBSOLETE_REPLICATED_DATABASE_SETTING(M, Bool, cluster_secure_connection, false) \
```

Comment:
The feature is still experimental, so we can simply remove these settings.

Reply:
Yep, it will not work anyway, because the settings are not used now, so let's remove

Reply:
Ok.